.github/actions/setup-rocm/action.yml

```
@@ -57,6 +57,26 @@ runs:
             exit 1
         fi
 
+    - name: Runner health check diskspace
+      if: always()
+      shell: bash
+      run: |
+        diskspace_cutoff=70
+        diskspace=$(df -H / --output=pcent | sed -n 2p | sed 's/%//' | sed 's/ //')
+        msg="Please file an issue on pytorch/pytorch reporting the faulty runner. Include a link to the runner logs so the runner can be identified"
+        if [[ $diskspace -ge diskspace_cutoff ]] ; then
+            docker system prune -af
+            diskspace_new=$(df -H / --output=pcent | sed -n 2p | sed 's/%//' | sed 's/ //')
+            if [[ $diskspace_new -gt diskspace_cutoff ]] ; then
```

Same as above (also, you are missing $ to dereference variable, aren't you?"
```suggestion
            if [[ "${diskspace_new}" -gt "${diskspace_cutoff}" ]]; then
```